dbt_project.yml

```diff
@@ -1,9 +1,9 @@
 config-version: 2
 
 name: 'asana_source'
-version: '0.0.1'
+version: '0.2.0'
 
-require-dbt-version: '>=0.17.0'
+require-dbt-version: '>=0.18.0'
```

Comment:
We should limit the required dbt version at <0.19.0

require-dbt-version: [">=0.18.0", "<0.19.0"]